Disher/Models/Recipe.m

```diff
@@ -27,4 +27,33 @@ + (Recipe *) initWithRecipe:(NSString *) name withURL:(NSString *) imgURL withSo
     return newRecipe;
 }
 
++ (void) getRecipeInfo:(NSString *)recipeID withSource:(NSString *) source withCompletion:(void(^)(NSDictionary *recipeInformation))completionHandler {
+    NSURL *url;
+    if ([source isEqualToString:@"Spoonacular"]) {
```

Comment:
i'd recommend to start creating constants for all these literal strings we have here and there. for sources, you could even create `ENUM`s. maybe have all these in a file something like `StringLiteralsHelpers` or similar.